packages/react-dom/src/__tests__/ReactDOMForm-test.js

```diff
@@ -414,4 +414,40 @@ describe('ReactDOMForm', () => {
     expect(nav).toBe('Navigate to: http://example.com/submit');
     expect(actionCalled).toBe(false);
   });
+
+  // @gate enableFormActions || !__DEV__
+  it('allows a non-react html formaction to be invoked', async () => {
+    let actionCalled = false;
+
+    function action(formData) {
+      actionCalled = true;
+    }
+
+    const root = ReactDOMClient.createRoot(container);
+    await act(async () => {
+      root.render(
+        <form
```

Comment:
If the form is not a React element is already handled by the event system itself.